lib/samples/match_viewpoint_of_geo_views/match_viewpoint_of_geo_views.dart

```diff
@@ -83,16 +79,39 @@ class _MatchViewpointOfGeoViewsState extends State<MatchViewpointOfGeoViews>
     return [
       // Add a map view to the widget tree and set a controller.
       Expanded(
-        child: ArcGISMapView(
-          controllerProvider: () => _mapViewController,
-          onMapViewReady: onMapViewReady,
+        child: GestureDetector(
+          behavior: HitTestBehavior.translucent,
+          onTapDown: (_) => setState(() {
+            _isMapViewInteraction = true;
+            _isSceneViewInteraction = false;
+          }),
+          onDoubleTapDown: (details) => setState(() {
```

Comment:
For consistency, `(details)` should be `(_)`. Also the same for the ArcGISSceneView's onTapDown and onDoubleTapDown callbacks.